src/Microsoft.AspNet.Razor/Generator/TagHelperCodeGenerator.cs

```diff
@@ -66,6 +66,7 @@ public override void GenerateStartBlockCode(Block target, CodeGeneratorContext c
 
                 attributes[attribute.Key] = new ChunkBlock
                 {
+                    Association = first?.Association,
```

Comment:
I c I c, dur definitely makes sense now that I think about it. It's good as is :+1: